library/src/test/java/com/getstream/sdk/chat/rest/core/ClientTest.java

```diff
@@ -951,6 +953,95 @@ void queryChannelWithoutIdSuccessTest() {
         verify(storage).insertMessagesForChannel(channel, response.getMessages());
     }
 
+    @Test
+    void watchChannelWithIdSuccessTest() {
```

Comment:
test is called `watchChannelWithIdSuccessTest` but calls and tests `queryChannel`, it does not make much sense :) Please rename + add a direct test for `channel.watch` and `channel.create` to make sure this is not creating regressions

Reply:
@tbarbugli , I have removed the problem code.
We have already `channel.watch` test case in `ChannelTest`

https://github.com/GetStream/stream-chat-android/blob/master/library/src/test/java/com/getstream/sdk/chat/models/ChannelTest.java#L112

Reply:
@tbarbugli, added `createTest`, either.